src/main/resources/swagger/api-docs.yaml

```diff
@@ -39,155 +39,269 @@ securityDefinitions:
-#          name: userEmail
-#          required: true
+#        - name: documentation
+#          in: query
+#          description: Documentation of method.
+#          required: false
 #          type: string
-#      tags:
-#        - Billing
-#      summary: add user to billing project the caller owns
-#      operationId: addUserToBillingProject
-#      security:
-#        - googleoauth:
-#            - openid
-#            - email
-#            - profile
-#    delete:
-#      responses:
-#        '200':
-#          description: Successfully Removed User From Billing Project
-#        '403':
-#          description: You must be a project owner to add a user to a billing project
-#        '404':
-#          description: User not found
-#        '500':
-#          description: Rawls Internal Error
-#      description: remove user from billing project the caller owns
-#      parameters:
-#        - in: path
-#          description: Project ID
-#          name: projectId
-#          required: true
+#        - name: owner
+#          in: query
+#          description: Owner of method.
+#          required: false
 #          type: string
-#        - in: path
-#          description: role of user for project
-#          name: role
-#          required: true
+#        - name: payload
+#          in: query
+#          description: Payload of method in WDL.
+#          required: false
 #          type: string
-#          enum: ["user", "owner"]
-#        - in: path
-#          description: User Email
-#          name: userEmail
-#          required: true
+#        - name: entityType
+#          in: query
+#          description: Type of Method - Task or Workflow
+#          required: false
 #          type: string
-#      tags:
-#        - Billing
-#      summary: remove user from billing project the caller owns
-#      operationId: removeUserFromBillingProject
-#      security:
-#        - googleoauth:
-#            - openid
-#            - email
-#            - profile
-
-  /api/configurations:
-    get:
+#        - name: excludedFields
+#          in: query
+#          description: Fields to exclude in JSON response.
+#          required: false
+#          type: string
+#        - name: includedFields
+#          in: query
+#          description: Fields to include in JSON response
+#          required: false
+#          type: string
+      responses:
+        '200':
+          description: An array of configurations.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ConfigurationResponse'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: Add a Method.
+      description: |
+        Inserts a method into the method repository and
+        gives the owner full access permissions. Must supply a namespace
+        name, payload, and entityType. Cannot supply a snapshotId
+      consumes:
```

Comment:
the operation object doesn't have a description field - but it does have a notes field which can be used for a more verbose explanation